### ISSUE TITLE
Some more not dark zones need CSS

### ISSUE THREAD

**Issue by Presskopp** [danieltj27/Dark-Mode#53]
1) Permalinks active tag:

```
button.button.button-secondary.active {
    background-color: transparent !important;
}
```

OR

```
button.button.button-secondary.active {
    background-color: #bbc8d4 !important;
}
```

2) Menu Settings:

```
div#post-body {
    background-color: #191F25 !important;
}
```

also we need something for "Delete Menu" button (which isn't a button)

3) Editor

Here we need something like

```
ul[role="tree"] {
    background-color: #191f25 !important;
}
```

4) Media Modal

open Attachment Details..

I'm unable to solve it.

**Comment by danieltj27** [comment]
Thanks for the report. I'll take a look at these.

**Comment by danieltj27** [comment]
I've fixed a couple of these issues, I'm punting some others to the next release though. Namely the attachment pop up which isn't styled at all. In fact, it doesn't seem to be affected by any of the Dark Mode CSS at all which is weird.

Will also need to go back to the permalink button as well. It looks fine for me - I can't replicate the issue of it not being styled.

I'll open a new issue regarding the attachment popup.

At least mostly fixed in commit 16b30b0.
